### PR TITLE
[2.3] Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | - |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,11 +1,33 @@
 Contributing
 ------------
 
-Symfony is an open source, community-driven project. If you'd like to contribute,
-please read the [Contributing Code][1] part of the documentation. If you're submitting
-a pull request, please follow the guidelines in the [Submitting a Patch][2] section
-and use the [Pull Request Template][3].
+Symfony2 is an open source, community-driven project.
```
Comment (xabbuh) [inline CONTRIBUTING.md]: In the documentation, we recently replaced Symfony2 everywhere with Symfony.

Comment (fabpot) [inline CONTRIBUTING.md]: Yep, no Symfony2 anymore :)

**Comment by stof** [comment]
:+1:

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

**Pull request merged**